Protocol sync updates (#394)
MukurtuCMS/mukurtu-user-manual: docs/media/CreateMediaAssets.md

## Locally hosted media

- Names are automatically generated from the filename of the media. 

> - *Cultural protocols* and *Sharing settings* are required. We have included an optional **Sync Protocols with content** setting that can be enabled when appropriate.

Reviewer: ```suggestion
- *Cultural protocols* and *Sharing settings* are required. See [Manage Media Access with Cultural Protocols](ManageMediaAccessWithProtocols.md) for more information.
```

## Remotely hosted media 

- Enter the URL to link to your media asset.  
- Filenames automatically generate for remotely hosted media.

> - Make sure you select a *Sharing Setting* and *Cultural Protocol* to apply to your media, or select the **Sync Protocols with content** setting. For more information on this setting, refer to [Manage Media Access with Cultural Protocols](ManageMediaAccessWithProtocols.md).

Reviewer: ```suggestion
- *Cultural protocols* and *Sharing settings* are required. See [Manage Media Access with Cultural Protocols](ManageMediaAccessWithProtocols.md) for more information.
```
